docker_hub/README.md

```diff
@@ -0,0 +1,10 @@
+Use the Adapt Docker image as follows:
+
+1. Pull the adapt image to your machine: `docker pull adapt:latest`
```

Comment:
It'll be a while before Adapt can show enough traction to get the Docker Hub "adapt" name. In the meantime, probably the right name/tag is `unboundedsystems/adapt`